extender/client.go

```diff
@@ -0,0 +1,25 @@
+package extender
+
+import (
+	"log"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func GetKubeClient(kubeConfig string) (kubernetes.Interface, *rest.Config, error) {
+	clientConfig, err := rest.InClusterConfig()
+	if err != nil {
+		log.Print("not in cluster - trying file-based configuration")
+		clientConfig, err = clientcmd.BuildConfigFromFlags("", kubeConfig)
+		if err != nil {
+			return nil, nil, err
+		}
+	}
+	kubeClient, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return nil, nil, err
+	}
+	return kubeClient, clientConfig, nil
```

Comment:
Good idea to include utility functions like this in the extender library. Once this is merged we can update the TAS code to use this and delete its own method.